src/Symfony/Component/String/Inflector/EnglishInflector.php

```diff
@@ -0,0 +1,490 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\String\Inflector;
+
+use Symfony\Component\String\AbstractString;
+
+/**
+ * @experimental in 5.1
```

Comment:
why experimental if it is code copied from the existing non-experimental component ?